integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/OpenConditionsSpecification.groovy

```diff
@@ -105,6 +107,44 @@ class OpenConditionsSpecification extends AbstractSpecification {
     sevenOrMoreYearsNo.click()
     submitButton.click()
 
+    then: 'the Victim Contact Scheme page is displayed'
+    at VictimContactSchemePage
+
+    when: 'I submit a blank page'
```

Comment:
It would be probably a conversation with Raza how much testing we need to add here. 
From the perspective of testing pyramid and of splitting DEV/QA work, tooo. 
For the other epics we only added the happy path I think.

Reply:
i have added since i wanted to test it in detail. Will speak to Raza but can we leave it for now?

Reply:
Yes, of course, but maybe going forward Raza will write these. And even he can skip it, if it is covered by unit tests.